Guix builds status

For the current main branch (2aa44e608ac70ed7e56994634d8c15cf192f8390) Guix builds for `riscv64-linux-gnu` and `x86_64-w64-mingw32` failed.

For other platforms:
```
$ find guix-build-$(git rev-parse --short=12 HEAD)/output/ -type f -print0 | env LC_ALL=C sort -z | xargs -r0 sha256sum
635b24cf8bd83be192f52f595f936fff42a84e9d556cdfb956f4c69bda20698c  guix-build-2aa44e608ac7/output/aarch64-linux-gnu/SHA256SUMS.part
6e8776d85ed710ebaac8d1af9da81506eb4e4fc8ad9205ef12ac71bee3ec149e  guix-build-2aa44e608ac7/output/aarch64-linux-gnu/bitcoin-2aa44e608ac7-aarch64-linux-gnu-debug.tar.gz
12cf8d96230f2db389055dcf5409593556bffb54e2bcd4cff645a7e6a730b90e  guix-build-2aa44e608ac7/output/aarch64-linux-gnu/bitcoin-2aa44e608ac7-aarch64-linux-gnu.tar.gz
294c451b8e2ab3951520d53da04788048ea2d73aea8ba801aba34f51a860faa5  guix-build-2aa44e608ac7/output/arm-linux-gnueabihf/SHA256SUMS.part
6cff26db488213fa1351cb58f6eda11ccf752d7255af104a4c4ed288eb35771c  guix-build-2aa44e608ac7/output/arm-linux-gnueabihf/bitcoin-2aa44e608ac7-arm-linux-gnueabihf-debug.tar.gz
0165da9cf4427a91f5532eb035a433d0c9951d17fc2bed2e9c0c85d61c2da361  guix-build-2aa44e608ac7/output/arm-linux-gnueabihf/bitcoin-2aa44e608ac7-arm-linux-gnueabihf.tar.gz
65ebbaea7efab7daf5450f5276c5983498ed618ab575f66c4cd4d0e386fb458d  guix-build-2aa44e608ac7/output/dist-archive/bitcoin-2aa44e608ac7.tar.gz
350262aa3c8ef2cc026543cf1e5ff79e5f7d8cf9ee79f0bbaea2d74127acfdc4  guix-build-2aa44e608ac7/output/powerpc64-linux-gnu/SHA256SUMS.part
5f94885145fcd472f20c578c5d3aa3e1d42b541d00adf253a4d25ed61a26338d  guix-build-2aa44e608ac7/output/powerpc64-linux-gnu/bitcoin-2aa44e608ac7-powerpc64-linux-gnu-debug.tar.gz
0d043e82e8246657ae1b0cde74809873c4a14c0bfa69637804ddb4b5fde58ae5  guix-build-2aa44e608ac7/output/powerpc64-linux-gnu/bitcoin-2aa44e608ac7-powerpc64-linux-gnu.tar.gz
50721a56d111742035082c34156fe2f3cadcc2c48ea71c93a5789ff5966b1afa  guix-build-2aa44e608ac7/output/powerpc64le-linux-gnu/SHA256SUMS.part
5275ec85f43948bd6c9d7cf1ac76eb17949168918858cf33ee1bbc278f3bb668  guix-build-2aa44e608ac7/output/powerpc64le-linux-gnu/bitcoin-2aa44e608ac7-powerpc64le-linux-gnu-debug.tar.gz
7d694d4efecbf99e2495873e5d24d0edc7680b609fbdbde7cfb0a0f9532f4d49  guix-build-2aa44e608ac7/output/powerpc64le-linux-gnu/bitcoin-2aa44e608ac7-powerpc64le-linux-gnu.tar.gz
0bdbf4b1a10c476524988ff5f16355638937252e88a2560ca5fdf0af1dfd643c  guix-build-2aa44e608ac7/output/x86_64-apple-darwin18/SHA256SUMS.part
e6df4a92a221729c18fcc1135ca86eb7b9732c6a61ab2ec276cd08401a38fa95  guix-build-2aa44e608ac7/output/x86_64-apple-darwin18/bitcoin-2aa44e608ac7-osx-unsigned.dmg
25f2ce901edc2a23d524180d8d099d10ce651660207a5bbbb4d387e96ceaac8d  guix-build-2aa44e608ac7/output/x86_64-apple-darwin18/bitcoin-2aa44e608ac7-osx-unsigned.tar.gz
ab3317d798ee0552e74bb6254e5abe64945b80cb48ed04f12c49ecca8d7cea6a  guix-build-2aa44e608ac7/output/x86_64-apple-darwin18/bitcoin-2aa44e608ac7-osx64.tar.gz
6afb5c24e7f48e86a6f9623371b2994dbf3fc17b9789f04a9b2d6b98f9b3b085  guix-build-2aa44e608ac7/output/x86_64-linux-gnu/SHA256SUMS.part
2caa3acb62852b56f542358e5f5ee8d669d1abe5c541146a5b4d817b388b91ed  guix-build-2aa44e608ac7/output/x86_64-linux-gnu/bitcoin-2aa44e608ac7-x86_64-linux-gnu-debug.tar.gz
52ba14e0cd44883135f39a144c6d301613420dfe227486aa9de45d1063e77c15  guix-build-2aa44e608ac7/output/x86_64-linux-gnu/bitcoin-2aa44e608ac7-x86_64-linux-gnu.tar.gz
```

Excerpt from the log for the `x86_64-w64-mingw32` platform:
```
make[6]: Entering directory '/bitcoin/depends/work/build/x86_64-w64-mingw32/qt/5.12.11-739936266ce/qtdeclarative/src/plugins/scenegraph/d3d12'
make -f Makefile.Release
make[7]: Entering directory '/bitcoin/depends/work/build/x86_64-w64-mingw32/qt/5.12.11-739936266ce/qtdeclarative/src/plugins/scenegraph/d3d12'
fxc.exe /nologo /E VS_VertexColor /T vs_5_0 /Fh vs_vertexcolor.hlslh shaders/vertexcolor.hlsl
make[7]: fxc.exe: No such file or directory
make[7]: *** [Makefile.Release:226: vs_vertexcolor.hlslh] Error 127
make[7]: Leaving directory '/bitcoin/depends/work/build/x86_64-w64-mingw32/qt/5.12.11-739936266ce/qtdeclarative/src/plugins/scenegraph/d3d12'
make[6]: Entering directory '/bitcoin/depends/work/build/x86_64-w64-mingw32/qt/5.12.11-739936266ce/qtdeclarative/src/plugins/qmltooling/packetprotocol'
make -f Makefile.Release
make[6]: *** [Makefile:38: release] Error 2
make[6]: Leaving directory '/bitcoin/depends/work/build/x86_64-w64-mingw32/qt/5.12.11-739936266ce/qtdeclarative/src/plugins/scenegraph/d3d12'
make[5]: *** [Makefile:43: sub-d3d12-make_first] Error 2
make[5]: Leaving directory '/bitcoin/depends/work/build/x86_64-w64-mingw32/qt/5.12.11-739936266ce/qtdeclarative/src/plugins/scenegraph'
make[4]: *** [Makefile:69: sub-scenegraph-make_first] Error 2
make[4]: *** Waiting for unfinished jobs....
```

--- a/depends/packages/qt.mk
+++ b/depends/packages/qt.mk
@@ -165,6 +165,7 @@ $(package)_config_opts_riscv64_linux = -platform linux-g++ -xplatform bitcoin-li
 $(package)_config_opts_s390x_linux = -platform linux-g++ -xplatform bitcoin-linux-g++
 
 $(package)_config_opts_mingw32 = -no-opengl
+$(package)_config_opts_mingw32 += -no-d3d12
 $(package)_config_opts_mingw32 += -no-dbus
 $(package)_config_opts_mingw32 += -xplatform win32-g++
 $(package)_config_opts_mingw32 += "QMAKE_CFLAGS = '$($(package)_cflags) $($(package)_cppflags)'"

